includes/API/Request.php

```diff
@@ -66,4 +76,54 @@ public function set_data( $data ) {
 	}
 
 
+	/**
+	 * Gets the number of times this request has been retried.
+	 *
+	 * @since 2.1.0-dev.1
+	 *
+	 * @return int
+	 */
+	public function get_retry_count() {
+
+		return $this->retry_count;
+	}
+
+
+	/**
+	 * Marks the request as having been retried.
+	 *
+	 * @since 2.1.0-dev.1
+	 */
+	public function mark_retry() {
+
+		$this->retry_count++;
+	}
+
+
+	/**
+	 * Gets the number of retries to attempt if told to do so by Facebook.
+	 *
+	 * @since 2.1.0-dev.1
+	 *
+	 * @return int
+	 */
+	public function get_retry_limit() {
+
+		return $this->retry_limit;
```

Comment:
Is this something we may want to filter?

Reply:
Sure! added with c0a2519d475e923039a91d082b8e9f3866f184df